pkg/management/api/grpc/roles.go

```diff
@@ -0,0 +1,33 @@
+package grpc
```

Comment:
i think we should move this file to internal/auth or internal/management, because its business logic

Reply:
moved